src/addressResolvers/farcaster.ts

```diff
@@ -0,0 +1,115 @@
+import { capture } from '@snapshot-labs/snapshot-sentry';
+import { graphQlCall, Address, Handle, FetchError, isSilencedError, isEvmAddress } from './utils';
+
+export const NAME = 'Farcaster';
+const FNAMES_API_URL = 'https://fnames.farcaster.xyz/transfers?name=';
+const NEYNAR_API_URL = 'https://api.neynar.com/v2/farcaster/user/';
+const API_KEY = 'NEYNAR_API_DOCS'; // add api key on .env
+
+interface User {
+  username: string;
+  verified_addresses: {
+    eth_addresses: string[];
+    sol_addresses: string[];
+  };
+  pfp_url: string;
+}
+
+interface UserDetails {
+  [address: string]: User[] | string;
+}
+
+async function fetchData(url, options = {}) {
+    const response = await fetch(url, { ...options, headers: { accept: 'application/json', api_key: API_KEY, ...options.headers } });
```

Comment:
can you use `axios` for http interactions ? This is the lib use throughout the codebase, as `fetch` is native to node only from node 18

Reply:
Erratum: let's continue with `fetch`. Can you add `cross-fetch` as a dependency, and use the `fetch` from it ?